### PR TITLE
Initial Version

### PR DESCRIPTION
**Pull request by nelsonic**
This PR is meant to help people get _started_ with elm. (It's _not_ meant to be a "_complete_" tutorial)
I want to get the _initial_ **_Why? What? Who?_** sections into a presentable state (_please review and correct where appropriate_) and then invite contributions in the ***How?*** section (_the actual content - useful part - of the repo!_)

### CODE REVIEW

**Comment by naazy** [comment]
@nelsonic Are you happy for me to contribute to this PR? I have some things I think I could add to your sign up form example

**Comment by nelsonic** [comment]
@naazy please go for it. (_I have a couple of other priorities I need to before I can`continue` with `elm-lang` notes ..._)

File: examples/SignupForm.elm [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+-- Html.Events module into the current namespace.
+
+import Html.Attributes exposing (id, type_, for, value, class)
+
+
+-- With this import we are only bringing a few specific functions into our
+-- namespace, specifically "id", "type_", "for", "value", and "class".
+
+
+view model =
```
Comment (3tty0n) [inline examples/SignupForm.elm]: I want type annotation here.
e.g.)
```elm
view : Model -> Html Msg
view model = ...
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "learn-elm",
+  "version": "0.0.1",
+  "description": "A quick intro tutorial for learning Elm",
+  "dependencies": {
+    "elm": "^0.17.1"
```
Comment (3tty0n) [inline package.json]: Why is this not elm 0.18.0 ?

Comment (nelsonic) [inline package.json]: @3tty0n _great_ question! It _will_ be by the time it's "_ready_". 😉 
(_apologies for taking so **long** to get this done; other work commitments..._)
Feel free to `git pull` on the branch and make any changes you feel appropriate and push back up on the branch/PR, I will pull before continuing my work... thanks!! ❤️

File: package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "learn-elm",
+  "version": "0.0.1",
+  "description": "A quick intro tutorial for learning Elm",
+  "dependencies": {
+    "elm": "^0.18.0"
```
Comment (3tty0n) [inline package.json]: 👍

**Comment by nelsonic** [comment]
@iteles / @naazy at _some_ point today, if you have time to read thru this _initial version_ please do. and if you're "_happy_" please merge! thanks!

Review/Corrections are _very much_ welcome from _everyone_ just don't want to _assign_ too many people to avoid _spamming_ people...! 👍

**Comment by iteles** [comment]
This is an amazing initial version! Merging it in so that more people can begin to contribute, noting @naazy's [comment above](https://github.com/dwyl/learn-elm/pull/14#issuecomment-263629796).

**Pull request merged**